specification/configuration.md

```diff
@@ -89,6 +90,50 @@ environment variables:
   If `SPLUNK_CONFIG` is defined then these options are not required but MAY be
   defined.
 
+### Kubernetes
+
+While Kubernetes supports container technology that can be configured using
+environment variables, package management solutions such as Helm charts and
+Operators require YAML-based configuration. As a result, Kubernetes package
+management solutions MUST support the following YAML configuration options:
+
+- `splunkCommon`
+  - `clusterName` ()          : [REQUIRED] Name of the cluster.
+  - `distro` ()               : Which distribution of Kubernetes is deployed.
```

Comment:
`platform` is needed here as well I believe

Reply:
We need platform, provider, and distro?? Are all required? Do we have a default for any?

Reply:
Oh sorry, I mean `provider`, not `platform`. We need `provider` and `distro`, both of them are optional